### PR TITLE
doc update: configure persistent storage for event producer

### PR DESCRIPTION
**Pull request by jzding**
Signed-off-by: Jack Ding <jackding@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,82 +17,125 @@
...
+```
+
+HTTP producer example
+```yaml
  - name: cloud-event-sidecar
           image: quay.io/redhat-cne/cloud-event-proxy
           args:
             - "--metrics-addr=127.0.0.1:9091"
             - "--store-path=/store"
             - "--transport-host=ptp-event-publisher-service.openshift-ptp.svc.cluster.local:9043"
```
Comment (aneeshkp) [inline README.md:75]: ptp-event-publisher-service-NODE_NAME.openshift-ptp.svc.cluster.local:9043"
or 
with first sub string of node name speperated by "." delimiter example 
ptp-event-publisher-service-cnfde22.openshift-ptp.svc.cluster.local:9043"

File: README.md [hunk 2]
```diff
@@ -17,82 +17,125 @@
...
 
 ## Creating Publisher
-#### Publisher JSON Example 
+### Publisher JSON Example 
 Create Publisher Resource: JSON request
 ```json
 {
   "Resource": "/east-edge-10/vdu3/o-ran-sync/sync-group/sync-status/sync-state",
-  "UriLocation": "http://localhost:8089/ack/event"
+  "UriLocation": "http://localhost:9085/ack/event"
```
Comment (aneeshkp) [inline README.md:138]: UriLocation should be consumer service api port which is request for event subscription , not 9085

File: README.md [hunk 3]
```diff
@@ -102,10 +145,12 @@ Create Publisher Resource: JSON response
   "Id": "789be75d-7ac3-472e-bbbc-6d62878aad4a",
   "Resource": "/east-edge-10/vdu3/o-ran-sync/sync-group/sync-status/sync-state",
   "UriLocation": "http://localhost:9090/ack/event" ,
```
Comment (aneeshkp) [inline README.md:147]: this is correct 9090

File: README.md [hunk 4]
```diff
@@ -17,82 +17,125 @@
...
+            - name: pubsubstore
+              mountPath: /store
+      volumes:
+        - name: pubsubstore
+          emptyDir: {}
+          persistentVolumeClaim:
+            claimName: cloud-event-proxy-store
+```
 
+Example PersistentVolumeClaim
```
Comment (aneeshkp) [inline README.md:100]: Don't you want PersistentVolume definition

File: README.md [hunk 5]
```diff
@@ -17,82 +17,125 @@
...
 HTTP consumer example
-```json
+```yaml
  - name: cloud-event-sidecar
           image: quay.io/redhat-cne/cloud-event-proxy
           args:
             - "--metrics-addr=127.0.0.1:9091"
             - "--store-path=/store"
             - "--transport-host=consumer-events-subscription-service.cloud-events.svc.cluster.local:9043"
             - "--http-event-publishers=ptp-event-publisher-service.openshift-ptp.svc.cluster.local:9043"
```
[the remark this answers is not in the document]
Comment (jzding) [reply README.md]: updated

File: README.md (hunk 2 above)
Comment (jzding) [reply README.md]: updated to 9090

File: README.md (hunk 4 above)
Comment (jzding) [reply README.md:103]: Customer is free to choose how the PersistentVolume is created. For example when the Local Storage operator is installed, PV can be created automatically by the Local Storage operator.

**Review by aneeshkp — APPROVED** [review]
/lgtm

**Pull request merged**